shiny/express/_page.py

```diff
@@ -0,0 +1,334 @@
+from __future__ import annotations
+
+from typing import Callable, Literal, Optional, cast
+
+from htmltools import Tag, TagAttrValue, TagChild, TagList
+
+from .. import ui
+from ..types import MISSING, MISSING_TYPE
+from ..ui._navs import NavMenu, NavPanel
+from ..ui._sidebar import Sidebar
+from ..ui.css import CssUnit
+from ._recall_context import RecallContextManager
+from ._run import get_top_level_recall_context_manager
+
+__all__ = (
+    "page_auto",
+    "page_auto_cm",
+    "set_page_title",
+    "set_page_lang",
+    "set_page_fillable",
+    "set_page_wider",
+    "use_page_fixed",
+    "use_page_fluid",
+    "use_page_fillable",
+    "use_page_sidebar",
+    "use_page_navbar",
+)
+
+
+def page_auto_cm() -> RecallContextManager[Tag]:
+    return RecallContextManager(
+        page_auto,
+        kwargs={
+            "_page_fn": None,
+            "_fillable": False,
+            "_wider": False,
```

Comment:
I don't feel strongly, but seems like slightly better naming?

```suggestion
            "_restrict_width": True,
```

Reply:
This ended up being `_full_width`